docs/book/component-gallery/image-builders/local.md

```diff
@@ -0,0 +1,47 @@
+---
+description: How to build container images locally
+---
+
+The local image builder is an [image builder](./image-builders.md) flavor which 
+comes built-in with ZenML and uses the local Docker installation on your client
+machine to build container images.
+
+{% hint style="info" %}
+ZenML uses the official Docker Python library to build and push your images. This library
+loads its authentication credentials to push images from the default config location: `$HOME/.docker/config.json`.
+If your Docker configuration is stored in a different directory, you can use the environment
+variable `DOCKER_CONFIG` to override this behavior:
+```shell
+export DOCKER_CONFIG=/path/to/config_dir
+```
+The directory that you specify here must contain your Docker configuration in a file called `config.json`.
+{% endhint %}
+
+## When to use it
+
+You should use the local image builder if:
+* you're able to install and use [Docker](https://www.docker.com) on your client machine.
+* you want to use remote components that require containerization without
+the additional hassle of configuring infrastructure for an additional component.
+
+## How to deploy it
+
+The local image builder comes with ZenML and works without any additional setup.
```

Comment:
I'll have to look into this in more detail, but it can probably be extracted to an integration at some point. However it's not really important IMO, the package is tiny anyway and doesn't conflict with anything.

Reply:
Discussion: for now we leave it as is, perhaps look into making this an integration in the future.